File: JANL.VB/Colors/Status.vb
Imports System.Drawing

Namespace Colors

    ''' <summary>
    ''' Status colors palette
    ''' </summary>
    ''' <remarks>
    ''' https://www.astrouxds.com/patterns/status-system/
    ''' </remarks>
    Friend Structure Status
        Public Shared ReadOnly Property Critical As Color = Color.FromArgb(255, 56, 56)
        Public Shared ReadOnly Property Serious As Color = Color.FromArgb(255, 179, 2)
        Public Shared ReadOnly Property Caution As Color = Color.FromArgb(252, 232, 58)
        Public Shared ReadOnly Property Normal As Color = Color.FromArgb(86, 240, 0)
        Public Shared ReadOnly Property Standby As Color = Color.FromArgb(45, 204, 255)
        Public Shared ReadOnly Property Off As Color = Color.FromArgb(158, 167, 173)

    End Structure

End Namespace

File: JANL.WinForms/Colors/Status.cs
using System;
using System.Drawing;
using JANL.Drawing;

namespace JANL.Colors
{
    /// <summary>
    /// Status colors palette
    /// </summary>
    /// <remarks>
    /// https://www.astrouxds.com/patterns/status-system/
    /// </remarks>
    [Obsolete(@"Use JANL.Drawing.AstroColor")]
    public struct Status
    {
#pragma warning disable CS1591 // Отсутствует комментарий XML для открытого видимого типа или члена
        public static Color Critical => AstroColor.Dark.Critical;
        public static Color Serious => AstroColor.Dark.Serious;
        public static Color Caution => AstroColor.Dark.Caution;
        public static Color Normal => AstroColor.Dark.Normal;
        public static Color Standby => AstroColor.Dark.Standby;
        public static Color Off => AstroColor.Dark.Off;
#pragma warning restore CS1591 // Отсутствует комментарий XML для открытого видимого типа или члена
    }
}